ui/rogue/presets.ts

```diff
@@ -271,13 +346,14 @@ export const DefaultIndividualBuffs = IndividualBuffs.create({
 	spiritOfZandalar: true,
 	valorOfAzeroth: true,
 	warchiefsBlessing: true,
+	spiritOfZandalar: true,
```

Comment:
This got screwed up in the merging but this just needs to be removed. It's already set above